docs/data/base/components/input/input.md

```diff
@@ -0,0 +1,58 @@
+---
+product: base
+title: React Input unstyled component and hook
+components: InputUnstyled
+githubLabel: 'component: input'
+packageName: '@mui/base'
+---
+
+# Unstyled input
+
+<p class="description">The `Input` component provides users with a field to enter and edit text.</p>
+
+Features:
+
+- ✨ Supports start and end adornments.
+- 🚀 Can be transformed to textarea using the `multiline` prop.
+- ♿️ Adds the appropriate ARIA roles automatically.
```

Comment:
Drop the periods in the list. Also, should `<textarea>` be formatted like an HTML tag? 

```suggestion
- ✨ Supports start and end adornments
- 🚀 Can be transformed to `<textarea>` using the `multiline` prop
- ♿️ Adds the appropriate ARIA roles automatically
```